src/styles/components/popup.css

```diff
@@ -147,3 +148,16 @@
   font-weight: 500;
   border: none;
 }
+.error-messages {
```

Comment:
1. Format
2. Error is not related popup

Reply:
I fixed @phuocduong-agilityio . Please help me check. Thanks !